x/liquidstakeibc/keeper/msg_server.go

```diff
@@ -504,6 +516,21 @@ func (k msgServer) LiquidStakeLSM(
 				)
 			}
 		}
+
+		ctx.EventManager().EmitEvents(sdktypes.Events{
+			sdktypes.NewEvent(
+				types.EventTypeLiquidStakeLSM,
+				sdktypes.NewAttribute(types.AttributeDelegatorAddress, delegator.String()),
+				sdktypes.NewAttribute(types.AttributeAmount, depositAmount.String()),
+				sdktypes.NewAttribute(types.AttributeAmountReceived, mintToken.Sub(protocolFee).String()),
+				sdktypes.NewAttribute(types.AttributePstakeDepositFee, protocolFee.String()),
+			),
+			sdktypes.NewEvent(
+				sdktypes.EventTypeMessage,
```

Comment:
i think from v47 onwards this one is not needed

Reply:
are we sure ?

it is present on all the events we emit, let's keep it for consistency and then do a full cleanup if needed

Reply:
Upgrading.md cosmos-sdk v47.0

`EventTypeMessage` events, with `sdk.AttributeKeyModule` and `sdk.AttributeKeySender` are now emitted directly at message excecution (in `baseapp`).
This means that the following boilerplate should be removed from all your custom modules:

```go
ctx.EventManager().EmitEvent(
	sdk.NewEvent(
		sdk.EventTypeMessage,
		sdk.NewAttribute(sdk.AttributeKeyModule, types.AttributeValueCategory),
		sdk.NewAttribute(sdk.AttributeKeySender, `signer/sender`),
	),
)
```